realm/realm-library/src/main/java/io/realm/RealmBaseRecyclerAdapter.java

```diff
@@ -0,0 +1,195 @@
+package io.realm;
```

Comment:
We have an outstanding issue for moving all Android related code to it's own package name: #1093 . So far we punted it because it was only one class, but when adding this, it will make a lot of sense to do that move now. So please move this to `io.realm.android`.